examples/cactus-check-connection-ethereum-validator/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "check-connection-ethereum-validator",
+  "version": "1.0.0-rc.3",
+  "private": true,
+  "scripts": {
+    "start": "node ../../dist/packages/cactus-cmd-socketio-server/src/main/typescript/routing-interface/www.js",
+    "debug": "nodemon --inspect ../../dist/packages/cactus-cmd-socketio-server/src/main/typescript/routing-interface/www.js",
+    "build": "npm run build-ts && npm run copy-static-assets && npm run copy-blp-config && npm run replace-blp-config-path",
+    "build-ts": "tsc -p ./tsconfig.json",
+    "tslint": "tslint -c tslint.json -p tsconfig.json './*.ts'",
+    "copy-static-assets": "ts-node copyStaticAssets.ts",
+    "copy-blp-config": "ts-node copyBLPConfig.ts",
+    "replace-blp-config-path": "ts-node replaceBLPConfigPath.ts",
+    "init-check-connection-ethereum-validator": "ln -s ../examples/cactus-check-connection-ethereum-validator/node_modules ../../dist/node_modules"
+  },
+  "dependencies": {
+    "@types/node": "^14.0.24",
+    "socket.io": "^2.0.4",
+    "ts-node": "8.9.1"
+  },
+  "devDependencies": {
+    "tslint": "6.0.0",
```

Comment:
@StepniowskiP Any reason why ESLint (which is already a dependency) could not be used instead of TSLint (which has been deprecated for years)

Reply:
I think I can remove this dependency

Reply:
@StepniowskiP That would be great, thank you very much in advance!

Reply:
@petermetz I have removed this dependency in latest commit.

Reply:
@StepniowskiP Thank you very much! Since the dependency itself is gone, would it make sense to also remove the `tslint.json` config file?

Reply:
@petermetz Sure, I have deleted it

Reply:
@StepniowskiP Great, thank you!